config/initializers/one_time_tasks_runner.rb

```diff
@@ -0,0 +1,27 @@
+# @description Configuration information for the OneTimeTasker::TasksRunner
+#
+# @file config/initializers/one_time_tasks_runner.rb
+
+if defined?(OneTimeTasker::TasksRunner)
+
+  OneTimeTasker::TasksRunner.configure do |config|
+
+    # This string is appended to each .rake filename
+    # if all of the tasks in the rake file ran successfully.
+    # Appending this to the filename helps to keep the rake file
+    # from being run again.
+    #
+    # Default is '.ran'
+    config.successful_rakefile_extension = '.ran'
```

Comment:
Should this be configurable or a fixed constant?

If the former, would changing this at some point in the future would cause the previously-run rake files to be attempted again?  (or, is the mere fact that the filename for each does _not_ end in `.rake` prevent that?)

Reply:
Only files that end in `.rake` will be attempted (read, in fact).  So any ending that is not `.rake` can be anything.

A fixed-constant makes sense for a default value, here.  The end user(s) should be able to specify what string to use, so that's a configuration variable.

Reply:
Just FYI: here's the method that gets the list of `.rake` files to evaluate:
`OneTimeTasker::TasksFinder one_time_rake_files`
https://github.com/AgileVentures/shf-project/pull/686/files#diff-ffa7a456562d2fad40916a1d3a0b5f06R264